frontend/src/components/dashboard/aliases/CategoryFilter.tsx

```diff
@@ -32,6 +32,11 @@ export type Props = {
   onChange: (selected: SelectedFilters) => void;
 };
 
+type OnCloseParams = {
```

Comment:
React newbie question: Why does this not have `export type` compared to the other two declarations above?

Reply:
It’s not necessary to export type `OnCloseParams` since it’s only used within this component. If a `type` is exported it can be imported from another file using a named import. The type `SelectedFilters` for example is imported and used by the component `AliasList` as well: https://github.com/mozilla/fx-private-relay/blob/main/frontend/src/components/dashboard/aliases/AliasList.tsx#L9.

Reply:
Thanks for the explanation! 🐣